scipy/stats/_unuran/unuran_wrapper.pyx

```diff
@@ -229,7 +229,7 @@ cdef dict _unpack_dist(object dist, str dist_type, list meths = None,
                     self.support = dist.support
                 def pdf(self, x):
                     # some distributions require array inputs.
-                    x = np.atleast_1d((x-self.loc)/self.scale)
+                    x = np.asarray((x-self.loc)/self.scale)
```

Comment:
I do not remember why we chose `atleast_1d` initially. when I added `logpdf` in a followup PR, I chose `asarray`.